Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -951,7 +951,7 @@ command scriptComment
    # OK-2009-03-03 : Bug 7537 - Reformat the handler after commenting, this may be flawed,
    # but the proper solution is rather time-consuming.
```

Comment:
perhaps.... unfortunately I don't know how OK thought it was flawed. There's probably something I'm missing there.